external/iotivity/iotivity_1.2-rel/resource/csdk/connectivity/lib/libcoap-4.1.1/LICENSE.GPL

```diff
@@ -1,280 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```

Comment:
If you want to remove this file, you can modify readme in same location.

Reply:
If LICENSE.GPL remains at Tizen RT code, it will be detected in the Blackduck-protex. Do not we need to delete LICENSE.GPL files if we have README just like so? Then, it would be better to add README to /external/include/protocols/libcoap. (It already exists in other folders)

Reply:
I'm not sure that which one is correct.
But I saw that all files in each folder have comments like "Please see README for terms of use."
and license information of README files is different.
README in external/libcoap has only BSD license information.
So I thaught if you delete this file, you should modify license information of README regarding it.
Let's check that we can just remove this file and modify README. if we choose one of dual license